aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -2687,6 +2687,11 @@ private void clusterMemberJoined(final int memberId, final ClusterMember[] newMe
         highMemberId = Math.max(highMemberId, memberId);
 
         final ClusterMember eventMember = ClusterMember.findMember(newMembers, memberId);
+        if (eventMember != null && eventMember.publication() == null)
```

Comment:
What if eventMember is null? What happens after the if block?

Reply:
In the context of this commit. The null would be added to the array and to the idMap. Which I'm adding a change for as we speak.